nimpy.nim

```diff
@@ -934,12 +985,20 @@ proc pyObjToNim[T](o: PPyObject, v: var T) {.inline.} =
     elif T is array:
         conversionTypeCheck(pyLib.PyList_Type)
         pyObjToNimArray(o, v)
+    elif T is JsonNode:
+        # above `ref` since `JsonNode` is a `ref` object
+        conversionTypeCheck(pyLib.PyDict_Type)
```

Comment:
Why only dict? It could be anything representable with JsonNode.

Reply:
Simply because I wanted to keep this PR "simple".